packages/block-library/src/archives/edit.js

```diff
@@ -47,9 +46,7 @@ export default function ArchivesEdit( { attributes, setAttributes } ) {
 					controls={ [ 'left', 'center', 'right' ] }
 				/>
 			</BlockControls>
-			<Disabled>
-				<ServerSideRender block="core/archives" attributes={ attributes } />
-			</Disabled>
+			<ServerSideRender block="core/archives" attributes={ attributes } />
```

Comment:
ping @talldan as not sure what was the reason for it to be disable originally.

Also Adding a "Needs Design Feedback" label as it can be seen as a "good" decision to disable interactions with this kind of blocks which are meant for interactivity in frontend first.

Reply:
Hey, it's disabled as interactions with the archives block in the editor can trigger a navigation away from Gutenberg.

I personally don't think the `Disabled` should be removed because of that.

It looks like Categories doesn't suffer from the same problem, interactions open in a new tab.

Reply:
So it looks like the behaviour is just as expected.

Feel free to close the PR.

Reply:
This PR could potentially be retargetted to add the `Disabled` behavior to the Category block if that's something we want to make consistent.

Reply:
We could use some reflection. Although I think it's more correct to altogether disable interactions on `ServerSideRender`-powered elements such as the Archives block, that dropdown may seem confusing to users.

Would it make any sense to add a new wrapper component like the following?

```js
<DisableEvents>
  <ServerSideRender … />
</DisableEvents>
```

`DisableEvents` could also be more specific, like `DisableRedirects`, etc. Or we could just iterate on `Disable` like so:

```js
<Disable allowClicks={ true } allowEvents={ false /* default */ }>
  <ServerSideRender … />
</Disable>
```

Reply:
We [discussed this briefly](https://wordpress.slack.com/archives/C02S78ZAL/p1554220997022000) in #design chat today. If we can allow people to open the dropdown, but prevent any events from triggering once they're inside it (in other words, mimicking what happens with the Categories dropdown now), that seems solid from a design perspective. 

Going to remove the `Needs Design Feedback` label for now.

Reply:
@kjellr This solution is not something we can "generalize" the fact that it works like that in the categories dropdown is "luck". Since with the server side rendering we don't control the output, the solution here is disable everything or just leave the result as is.

Reply:
>  the fact that it works like that in the categories dropdown is "luck".

😄 Got it, thanks for clarifying. I think it makes sense to be consistent in any case, so perhaps we need to just disable everywhere then.